stanford/roles/deployment/tasks/main.yml

```diff
@@ -34,9 +34,9 @@
     cidr: "{{ CLUSTER_NAT_SUBNET.cidr }}"
     az: "{{ CLUSTER_NAT_SUBNET.zone }}"
     resource_tags:
-      Cluster: nat
+      Cluster: public
```

Comment:
I actually can't remember that conversation but it being 'nat' is good with me also.

Reply:
The "noun vs. adjective" debate aside, I think calling it `nat` is objectively worse.
Putting a public ELB into a `nat` subnet, vs. putting a public `elb` into a `public` subnet, seems even more opaque and confusing/misleading.

As mentioned, I think the optimal approach would be to fully split out the two (`nat` and `public`), but that requires additional work, as the resources are already co-mingled. I'm open to addressing this work in the future, but i don't think we should be moving unrelated ELBs at this time. The only reason I waded in originally was because the bug prevented the jenkins/build deploy from working properly.

And to reiterate, this is already referred to as `public` and _not_ `nat` throughout each production deployment; I was just updating the code that generates this tag to match the value the tags already had in prod.

You're right that intermingling the terminology can be confusing, so I've gone ahead and pushed up an additional commit that renames any additional references to `nat` (except "Create NAT gateway") to say `public`.